### PR TITLE
customQuotes/quotes.json

### PR DESCRIPTION
**Pull request by RS-labhub**
Added a new quote

### CODE REVIEW

**Comment by RS-labhub** [comment]
@tincho4t please review this

File: customQuotes/quotes.json [hunk 1]
```diff
@@ -43,4 +43,8 @@
         "quote":"It's not a bug it's a feature",
         "author":"Unknown"
     }
+     {
+        "quote":"Of course, bad code can be cleaned up. But it’s very expensive.",
+        "author":"Robert C. Martin"
+    }
```
Comment (shravan20) [inline customQuotes/quotes.json:49]:
```suggestion
     ,{
        "quote":"Of course, bad code can be cleaned up. But it’s very expensive.",
        "author":"Robert C. Martin"
    }
```


Please format the json as well

**Review by shravan20 — CHANGES_REQUESTED** [review]

**Comment by RS-labhub** [comment]
@shravan20 Done the suggested changes.

**Comment by shravan20** [comment]
Please format the json

**Comment by shravan20** [comment]
@RS-labhub : Please format the json

**Pull request closed without merging**